### ISSUE TITLE
Missing git tag

### ISSUE THREAD

**Issue by tantale** [vrcmarcos/python-deprecated#1]
Your library should be tagged.

Could you, please, add the tag "v1.0.0" ?

**Comment by vrcmarcos** [comment]
Sorry, @tantale ! You're right, but I haven't had enough time to take care of this project. I've added you as a colaborator. Could you help me with this?

**Comment by vrcmarcos** [comment]
Oh, I just saw that I already had published the 1.0.0 tag hahaha!

**Comment by tantale** [comment]
Usually, we put a "v" in front of version number to name a tag. So, you could rename to "v1.0.0".

To rename a tag:

```bash
git tag new old
git tag -d old
git push origin :refs/tags/old
git push --tags
```

Yes, I can help you (there is no much things to do so).

What I have in mind:

- add a [bumpversion](https://pypi.python.org/pypi/bumpversion) configuration file to bump your version automatically (increment the version number and add a tag).
- replace the Makefile with a [tox](https://tox.readthedocs.io/en/latest/) configuration to be compatible with Windows.
- improve the documentation
- register into [ReadTheDocs](https://readthedocs.org)
- register into [AppVeyor](https://www.appveyor.com)
- make some recommendations on SO to encourage developers to use your/our work.

Regards,
– Laurent.
